src/pages/Home.jsx

```diff
@@ -0,0 +1,114 @@
+// pages/Home.jsx
+import styled from 'styled-components'
+import { useState } from 'react'
+import { useTodoStore } from '../stores/TodoStore'
+import { TodoInput } from '../components/TodoInput'
+import { TodoList } from '../components/TodoList'
+import Lottie from 'lottie-react'
+import todoAnimation from '../assets/todo-animation.json'
```

Comment:
Minor thing, but your code looks so good, so I don't have much to comment on 😉 

You can order your imports like this:
```
import { useState } from 'react'
import Lottie from 'lottie-react'
import styled from 'styled-components'
import todoAnimation from '../assets/todo-animation.json'
import { useTodoStore } from '../stores/TodoStore'
import { TodoInput } from '../components/TodoInput'
import { TodoList } from '../components/TodoList'
```
with external libraries first (with react hooks first for clarity, but the others in albhabetical order).
Secondly assets, then stores, followed by components (in alphabetical order)